packages/dapp/src/hooks/useKey.ts

```diff
@@ -0,0 +1,55 @@
+import type { KeyRecord } from '../store/actions';
+import type { JsonRpcSigner } from '@ethersproject/providers';
+import { useState, useMemo, useEffect } from 'react';
+import { findKeyById, findKeyByTag } from './useKeysManager';
+import { useAppState } from '../store';
+import Logger from '../utils/logger';
+
+// Initialize logger
+const logger = Logger('useKeysManager');
+const UNKNOWN_ERROR = 'Unknown useKeysManager error';
+
+export type UseKeyHook = [
+  key: KeyRecord | undefined,
+  signer: JsonRpcSigner | undefined,
+  error: string | undefined,
+];
+
+// UseKey react hook
+export const useKeysManager = (
```

Comment:
here is the typo. The hook must be named `useKey`

Reply:
:heavy_check_mark: